static/app/utils/replays/hooks/useReplayData.tsx

```diff
@@ -240,8 +241,12 @@ function useReplayData({
       .concat(extraErrorPages)
       .concat(platformErrorPages)
       .flatMap(page => page.data);
+
```

Comment:
need this to return a count of errors not including UF for our count in the top right